recipe/meta.yaml

```
@@ -17,8 +17,14 @@ source:
     # Guard deprecated nanohttp cleanup for libxml2 2.14 where HTTP may be disabled
     - 0004-guard-nanohttp-cleanup.patch
 
+{% set build = 17 %}
+{% if license_family == "gpl" %}
+  {% set build = build + 100 %}
```

Let's explain this in a comment:
```suggestion
  # gpl gets a higher build number since we want to favor the build where librttopo is included
  {% set build = build + 100 %}
```